web/beacon-app/src/features/teams/routes/TeamsPage.tsx

```diff
@@ -14,31 +15,34 @@ export function TeamsPage() {
   const onClose = () => setIsModalOpened(false);
   const onOpen = () => setIsModalOpened(true);
 
+  const hasPermissions = hasPermission(USER_PERMISSIONS.COLLABORATORS_ADD);
+
   return (
     <AppLayout>
       <Heading as="h1" className="mb-4 text-lg font-semibold">
-        Team
+        <Trans>Team</Trans>
       </Heading>
       <p className="my-3 text-sm">
-        Add team members to collaborate on your projects. Team members have access to projects
-        across the organization.
+        <Trans>
+          Add team members to collaborate on your projects. Team members have access to projects
+          across the organization.
+        </Trans>
       </p>
       <div>
         <div className="flex justify-between rounded-lg bg-[#F7F9FB] px-3 py-2">
           <div className="flex items-center gap-3"></div>
           <div>
-            {hasPermission(USER_PERMISSIONS.COLLABORATORS_ADD) && (
-              <Button
-                data-cy="add-team-member"
-                className="flex items-center gap-1"
-                size="medium"
-                variant="primary"
-                onClick={onOpen}
-              >
-                <Union className="fill-white" />
-                Team Member
-              </Button>
-            )}
+            <Button
+              data-cy="add-team-member"
+              className="flex items-center gap-1"
+              size="medium"
+              variant="primary"
+              disabled={!hasPermissions}
+              onClick={onOpen}
+            >
+              <Union className="fill-white" />
+              <Trans>Team Member</Trans>
+            </Button>
```

Comment:
To keep consistency across all the apps, we should disable the button when a user does not have the right permission. this will fix the width background issue